core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -283,6 +341,42 @@ oppia.controller('ExplorationEditor', [
       });
     };
 
+    $scope.showWelcomeExplorationModal = function() {
+      $scope.welcomeModalIsOpening = true;
+      var startTutorial = $scope.startTutorial;
+      var modalInstance = $modal.open({
+        templateUrl: 'modals/welcomeExploration',
+        backdrop: true,
+        controller: [
+          '$scope', '$modalInstance', function($scope, $modalInstance) {
+            $scope.beginTutorial = $modalInstance.close;
+
+            $scope.cancel = function() {
+              $modalInstance.dismiss('cancel');
+            };
+          }],
+        windowClass: 'oppia-welcome-modal'
+      });
+
+      modalInstance.result.then(function() {
+      // Called after $modal.close()
+      startTutorial();
+    }, function() {
+      // Called after $modalInstance.dismiss
+      stateEditorTutorialFirstTimeService.markTutorialFinished();
```

Comment:
Three console errors are actually emitted here. I traced the problem down to the $scope.$apply() in EditorNavigation, under the comment "Without this, the popover does not trigger". It looks like that comment is obsolete, and the $scope.$apply() can be removed (but please double check this, and make sure that no console errors occur in either case). Thanks!